packages/eslint-config-khan/index.js

```diff
@@ -72,30 +69,6 @@ module.exports = {
         "template-curly-spacing": OFF, // TODO(kevinb): enable this
         "valid-jsdoc": OFF, // TODO(kevinb): Enable this since we are using jdocs in some places.
 
-        /**
```

Comment:
Are there TS eslint rules we want to be enabling?

Reply:
Yes, the migration guide has a mapping from Flow rules to TypeScript rules.  I'll add those to the config in the next PR when I enable eslint to check .ts files.

Reply:
Great, ok!